resop.mod

```diff
@@ -0,0 +1,27 @@
+set V; # set of graph nodes
```

Comment:
`V` in the LaTeX means the vertex operator of a graph. For example, V(G) would be the vertices of hardware graph G. While V(S_i) denotes the vertices (VNFs) of the service graph S_i.

My suggestion is to name the set of hardware nodes as `set N;`, for each hardware node is denoted as n_i.